### DISCUSSION TITLE
Insert contents of another note inline in Template?

### DISCUSSION THREAD

**Discussion by gazoir** [SilentVoid13/Templater#1094] [Help]
Hi I’m struggling to work out if it’s possible for Templater to look for content in another note and insert that into a template. 

The use case: I create a new daily note every day, and would like to create a different set of recurring tasks for different days. Ideally I would have a separate ‘Recurring Tasks’ note for each day of the week, and pull that day’s recurring note when I run my daily template first thing.

I’d be looking for something that did `tR =+ filepath/filename` and dumped the whole contents of the file out. Is this possible without the Obsidian API?

**Comment by Zachatoo** [comment]
Look for `tp.file.include` in the Templater docs, you can use it to bring in the content of other files.

**Reply by gazoir** [reply]
Thanks so much! That's working exactly as intended. I really appreciate the help.
